content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -136,6 +137,33 @@ done
 
 Once you successfully upgrade to version v1.6.0, you will be able to view information about engine image versions on the UI.
 
+### Danger Zone Setting Configuration
+
+Starting with Longhorn v1.6.0, Longhorn allows you to promptly update the [Danger Zone settings](https://longhorn.io/docs/1.6.0/references/settings/#danger-zone) without the need to wait for all volumes detached. This enhancement is designed to provide users with more flexibility and efficiency in managing their configurations. To achieve the immediate effect, one of the following conditions must be met:
+
+- No Attached Volumes: If there are no volumes attached, the setting change will take effect immediately.
+- Engine Image Upgrade (Volume Live Upgrade): During a volume live upgrade, which involves creating a new instance manager pod, the setting change will take effect immediately for the new pod.
+
+If immediate effect conditions aren't met, the setting change is reconciled hourly, taking effect when all volumes are detached.
+
+> **Tip:**
+> For best results, detach all volumes before configuring the Danger Zone settings.
+
+- When all volumes are detached, the settings in the following table are immediately applied and the system-managed components are restarted. If you do not detach all volumes before the settings are synced, the settings are not applied and you might need to reconfigure the settings after detaching the remaining volumes.
+
+  | Setting | Additional Information| Affected Components |
+  | --- | --- | --- |
+  | [Kubernetes Taint Toleration](../../references/settings/#kubernetes-taint-toleration)| [Taints and Tolerations](../../advanced-resources/deploy/taint-toleration/) | System-managed components |
+  | [Priority Class](../../references/settings/#priority-class) | [Priority Class](../../advanced-resources/deploy/priority-class/) | System-managed components |
+  | [System Managed Components Node Selector](../../references/settings/#system-managed-components-node-selector) | [Node Selector](../../advanced-resources/deploy/node-selector/) | System-managed components |
+  | [Storage Network](../../references/settings/#storage-network) | [Storage Network](../../advanced-resources/deploy/storage-network/) | Instance Manager and Backing Image components |
+  | [V1 Data Engine](../../references/settings/#v1-data-engine) || Instance Manager component |
+  | [V2 Data Engine](../../references/settings/#v2-data-engine) | [V2 Data Engine (Preview Feature)](../../v2-data-engine/) | Instance Manager component |
+  | [Guaranteed Instance Manager CPU](../../references/settings/#guaranteed-instance-manager-cpu) || Instance Manager component |
+  | [Guaranteed Instance Manager CPU for V2 Data Engine](../../references/settings/#guaranteed-instance-manager-cpu-for-v2-data-engine) || Instance Manager component |
+
```

Comment:
Suggest adding a sentence before the table and tell readers the purpose of the table.

Reply:
Added a sentence before the table and tell readers the purpose of the table.
Thanks.